extensions/adobe/experience/campaign/experienceevent.schema.json

```diff
@@ -0,0 +1,259 @@
+          "title": "Control Group Member",
+          "type": "boolean",
+          "description":
+            "The recipient of this message is a member of a control group for this campaign activity. Messages to control group members are not delivered.",
+          "default": false
+        },
+        "https://ns.adobe.com/experience/campaign/message/testMember": {
+          "title": "A/B Test Member",
+          "type": "boolean",
+          "description":
+            "The recipient of this message is a member of a test group for this campaign activity.",
+          "default": false
+        },
+        "https://ns.adobe.com/experience/campaign/message/size": {
+          "title": "Size (Bytes)",
+          "type": "integer",
+          "description":
+            "Size in bytes of the message.\n* For email, the `size` reflects the all MIME envelop of the message, including the encoded attachments if any.\n* For SMS, the `size` reflects the number of bytes necessary to deliver the message.\nIf that number is over 160 the message will be delivered over more than one SMS.",
+          "minimum": 0,
+          "maximum": 2147483647
+        },
+        "https://ns.adobe.com/experience/campaign/message/reason": {
+          "title": "Reason",
+          "type": "integer",
+          "meta:enum": {
+            "Undefined": "Not defined",
+            "UnknownUser": "User unknown",
+            "InvalidDomain": "Invalid domain",
+            "Unreachable": "Unreachable",
+            "Disabled": "Account disabled",
+            "MailboxFull": "Mailbox full",
+            "NotConnected": "Not connected",
+            "Refused": "Refused",
+            "ErrorIgnored": "Error ignored",
+            "AddressUndefined": "Address not specified",
+            "Blacklisted": "Blacklisted address",
+            "Quarantine": "Quarantined address",
+            "Duplicate": "Duplicate",
+            "TypologyRule": "Excluded by a SQL rule",
+            "BusinessRanking": "Excluded after arbitration",
+            "Cancelled": "Delivery canceled",
+            "Quality": "Bad-quality address",
+            "Unchecked": "Unqualified address",
+            "OfferMissing": "Not eligible for the offers",
+            "OverDeliveryLimit": "Target limited in size",
+            "Expired": "Expired",
+            "TooLong": "Text too long",
+            "Untranslatable": "Character not supported by encoding",
+            "ControlGroup": "Control address"
+          },
+          "description":
+            "The reason why the message could/would not be delivered.",
+          "default": "Undefined"
+        },
+        "https://ns.adobe.com/experience/campaign/message/reasonMessage": {
+          "title": "Reason Message",
+          "type": "string",
+          "description":
+            "Reason of message delivery failure when the status is `bounce` or `nonDeliverable`."
+        },
+        "https://ns.adobe.com/experience/campaign/message/ip": {
```

Comment:
shouldn't this be outboundIP, to be more descriptive?

Reply:
Agree.